docs/openstack/create-and-connect-to-the-VM/create-a-Windows-VM.md

```diff
@@ -0,0 +1,216 @@
+    | 8c87cf6f-32f9-4a4b-91a5-0d734b7c9770 | ubuntu-20.04-x86_64 | active |
+    | da314c41-19bf-486a-b8da-39ca51fd17de | ubuntu-22.04-x86_64 | active |
+    +--------------------------------------+---------------------+--------+
+
+In the example below, this is image id `da314c41-19bf-486a-b8da-39ca51fd17de` for
+`ubuntu-22.04-x86_64`.
+
+Creating a disk from this image with a size of **80GB** is as follows.
+
+    openstack volume create --image da314c41-19bf-486a-b8da-39ca51fd17de --size 80 --description "Using Ubuntu Image" my-volume
+    +---------------------+--------------------------------------+
+    | Field               | Value                                |
+    +---------------------+--------------------------------------+
+    | attachments         | []                                   |
+    | availability_zone   | nova                                 |
+    | bootable            | false                                |
+    | consistencygroup_id | None                                 |
+    | created_at          | 2024-02-03T23:38:50.000000           |
+    | description         | Using Ubuntu Image                   |
+    | encrypted           | False                                |
+    | id                  | d8a5da4c-41c8-4c2d-b57a-8b6678ce4936 |
+    | multiattach         | False                                |
+    | name                | my-volume                            |
+    | properties          |                                      |
+    | replication_status  | None                                 |
+    | size                | 80                                   |
+    | snapshot_id         | None                                 |
+    | source_volid        | None                                 |
+    | status              | creating                             |
+    | type                | tripleo                              |
+    | updated_at          | None                                 |
+    | user_id             | 938eb8bfc72e4cb3ad2b94e2eb4059f7     |
+    +---------------------+--------------------------------------+
+
+Checking the status again using `openstack volume show my-volume` will allow the
+volume creation to be followed.
+
+"downloading" means that the volume contents is being transferred from the image
+service to the volume service
+
+"available" means the volume can now be used for booting. A set of volume_image
+meta data is also copied from the image service.
+
+### Launch instance from existing bootable volume
+
+### 1. Using Horizon dashboard
+
+Navigate: Project -> Volumes -> Volumes.
+
+Once successfully Volume is created, we can use the Volume to launch an instance
+as shown below:
+
+![Launch Instance from Volume](images/launch_instance_from_volume.png)
+
+!!! note "Note"
+    Only one instance at a time can be booted from a given volume. Make sure
+    'Delete Volume on Instance Delete' is selected "No" if you want the volume
```

Comment:
is selected "No" -> has a check in the "No" check box
This seems like an image would be useful to show what selecting no looks like.